core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1071,10 +1072,20 @@ private void pauseIfPauseOnDisconnect() {
         }
     }
 
-    private void unpauseIfPauseOnDisconnect() {
+    /**
+     * @param bluetooth true if the event for unpausing came from bluetooth
+     */
+    private void unpauseIfPauseOnDisconnect(boolean bluetooth) {
         if (transientPause) {
             transientPause = false;
-            if (UserPreferences.isPauseOnHeadsetDisconnect() && UserPreferences.isUnpauseOnHeadsetReconnect()) {
```

Comment:
I got rid of the check for isPauseOnHeadsetDisconnect because:

1. That's enforced by the UI and,
2. Maybe that's what the user wants anyways?